components/integrations/installed_incoming_webhook.tsx

```diff
@@ -135,14 +136,15 @@ export default class InstalledIncomingWebhook extends React.PureComponent<Props>
             );
         }
 
-        let actions = null;
-        if (this.props.canChange) {
-            actions = (<div className='item-actions'>
+        let actions = (this.props.canChange ?
+            <div className='item-actions'>
```

Comment:
please wrap multiline JSX in parentheses (not the whole condition):
```suggestion
        let actions = this.props.canChange ? (
            <div className='item-actions'>
```